### PR TITLE
Dev UI modifiting footer header

### PR DESCRIPTION
**Pull request by IvanPadalka**
modifiting footer header, google maps etc

### CODE REVIEW

File: src/main/resources/templates/about/footer.ftl [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+        display: inline !important;
+        padding: 0 !important;
+        margin: 0 !important;
+    }
+</style>
+<style type="text/css" data-type="vc_shortcodes-custom-css">
+    .vc_custom_1472631590873 {
+        background-image: url(https://bridge82.qodeinteractive.com/wp-content/uploads/2016/08/image.jpg?id=22) !important;
+        background-position: center !important;
+        background-repeat: no-repeat !important;
```
Comment (MarrMika) [inline src/main/resources/templates/about/footer.ftl:262]: Gather all the styles into one <style> tag

File: src/main/resources/templates/about/header.ftl [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script>
+    <script src="https://bridge82.qodeinteractive.com/wp-includes/js/wp-emoji-release.min.js?ver=5.3" type="text/javascript" defer=""></script>
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/7de6a.css" media="all">
+    <link rel="stylesheet" id="qode_demos_css-css" href="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/css/qode-demos-modules.css?ver=5.3" type="text/css" media="all">
+    <link rel="stylesheet" id="qode_demos_swiper_css-css"  href="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/css/swiper.css?ver=5.3" type="text/css"  media="all">
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/c138f.css" media="all">
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/2bf8a.css" media="all">
+    <script src="https://bridge82.qodeinteractive.com/wp-content/cache/minify/129fc.js"></script>
+</head>
```
Comment (MarrMika) [inline src/main/resources/templates/about/header.ftl:27]: Do we really need all these libraries?

**Review by MarrMika — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/about/header.ftl (hunk 2 above)
Comment (IvanPadalka) [reply src/main/resources/templates/about/header.ftl]: Deleted unused libraries

File: src/main/resources/templates/about/footer.ftl (hunk 1 above)
Comment (IvanPadalka) [reply src/main/resources/templates/about/footer.ftl:262]: Gathered

**Review by MarrMika — APPROVED** [review]
Good job!

**Pull request merged**